src/Model/Lumberjack.php

```diff
@@ -119,12 +120,17 @@ public function stageChildren($showAll = false)
         return $staged;
     }
 
+    protected function updateGetChildrenForTree(mixed $includeDeleted, SS_List &$children): void
+    {
+        $children = $this->excludeSiteTreeClassNames($children);
+    }
+
     /**
      * Excludes any hidden owner subclasses. Note that the returned DataList will be a different
      * instance from the original.
      *
-     * @param DataList $list
-     * @return DataList<SiteTree>
+     * @param SS_List $list
+     * @return SS_List<SiteTree>
```

Comment:
This has to change since we're passing an `ArrayList` from `updateGetChildrenForTree()`.
Thankfully no actual functionality relied on it being an explicit `DataList`.